### ISSUE TITLE
Light Spectrum final check

### ISSUE THREAD

**Issue by terracoda** [phetsims/molecules-and-light#340]
We describe the Light Spectrum Diagram in detail. However some of the numbers are rounded up or down. I am wondering if this is a mistake or if it is ok to approximate.?

Here's the Diagram
![Screen Shot 2020-05-12 at 3 22 33 PM](https://user-images.githubusercontent.com/1902116/81728067-6deb5080-9464-11ea-9001-c0077925e79b.png)

Notice that some of the dotted lines creating the demarcations fall between the ticks, thus falling on a half number. All the descriptions, however, describe whole numbers.

**For example:**
- Radio Wave - Wavelengths greater than 10 to the 4 meters to 5 times 10 to the negative 1 meters.
- Microwave - Frequencies 10 to the 9 to 5 times 10 to the 11 Hertz.

There are more examples.
@BLFiedler, can you double check the numbers to make sure they are accurate.

**Comment by terracoda** [comment]
@BLFiedler, you can use the A11y View to read it all at a bigger font
![Screen Shot 2020-05-12 at 3 32 17 PM](https://user-images.githubusercontent.com/1902116/81728985-d850c080-9465-11ea-8993-fae1833ddfee.png)

**Comment by terracoda** [comment]
@arouinfar, @BLFiedler gave it his best shot, can you verify the ranges in the descriptions to make sure they are close enough to the visual as they can be or at least not incorrect.

**Comment by terracoda** [comment]
@BLFiedler said on slack:

> I used an external source and made the less confusing choice between the graphic and the external list of ranges
> I think I added the source into the description doc
> So, the external source said radio waves went to 5E-1

@arouinfar, this is a pixel-polish for the descriptions, if you will allow me to say that. If it sounds good to you, we are good.

**Comment by BLFiedler** [comment]
Looks like I used this (https://www.siyavula.com/read/science/grade-10/electromagnetic-radiation/11-electromagnetic-radiation-03) while generating the descriptions when I was a little unclear on the graphic.

**Comment by terracoda** [comment]
@arouinfar, there is also a summary before the list of frequencies and wavelengths which I did not include in the screen shot above. The full description can be accessed in the A11y View.

**Comment by arouinfar** [comment]
@terracoda @BLFiedler the descriptions all sound reasonable to me.

**Comment by terracoda** [comment]
@BLFiedler, I love the uses list for rays in that resource you linked to https://github.com/phetsims/molecules-and-light/issues/340#issuecomment-628117599
